### DISCUSSION TITLE
VSCode + Duckly on VPS (Create website hosted on VPS)

### DISCUSSION THREAD

**Discussion by darioZIP** [coder/code-server#6344] [Q&A]
Hello everyone. I tried looking around, but I'm kind of a noob in this kinda stuff so would like your feedback.
I am doing a UNI project with a colleague of mine. We have to create a Password Manager website.

Would it be possible to host Coder, with VSCode + Duckly (for collaboration) and see in real time the website while we're working on it? I can get a custom domain name too, since I saw it is suggested 
Thank you for your time.

**Comment by code-asher** [comment]
I have not tried it but I have heard others have used Duckly with code-server successfully.

We have some documentation here https://coder.com/docs/code-server/latest/collaboration
